llvm/test/CodeGen/RISCV/rvv/ctlz-sdnode.ll

```diff
@@ -1229,22 +1229,6 @@ define <vscale x 1 x i64> @ctlz_nxv1i64(<vscale x 1 x i64> %va) {
 ; RV64I-NEXT:    vsrl.vx v8, v8, a0
 ; RV64I-NEXT:    ret
 ;
-; CHECK-F-LABEL: ctlz_nxv1i64:
```

Comment:
Why are these lines deleted without any other lines being changed?

Reply:
oops, I forgot to modify the prefix, thank you.